### PR TITLE
do not allow to update global clusterset

### PR DESCRIPTION
**Pull request by ldpliu**
Signed-off-by: ldpliu <daliu@redhat.com>
https://github.com/stolostron/backlog/issues/24286

### CODE REVIEW

File: pkg/webhook/clusterset/validatingWebhook.go [hunk 1]
```diff
@@ -81,13 +81,21 @@ func (a *AdmissionHandler) validateResource(request *v1.AdmissionRequest) *v1.Ad
 			}
 			return status
 		}
-		//allow create/update legacy clusterset
-		if clusterset.Spec.ClusterSelector.SelectorType == clusterv1beta1.LegacyClusterSetLabel {
+		//allow create/update global clusterset
+		if clusterset.Name == clustersetutils.GlobalSetName {
```
Comment (qiujian16) [inline pkg/webhook/clusterset/validatingWebhook.go:85]: I think you have the validation on create, can we reuse the same logic?

Comment (ldpliu) [reply pkg/webhook/clusterset/validatingWebhook.go:85]: > I think you have the validation on create, can we reuse the same logic?

The logic here is used for both creating and updating a set.

Comment (ldpliu) [reply pkg/webhook/clusterset/validatingWebhook.go:85]: > if so, the message seems incorrect when create?

Update the message.

**Pull request merged**